src/bridges.ts

```diff
@@ -0,0 +1,43 @@
+export interface Bridge {
+  id: string
+  name: string
+  logo: string
+  aggregator_address: string
+  lim: string
+  affiliate: string
+  fee: number
+}
+
+export const THORSWAP: Bridge = {
+  id: 'thorswap',
+  name: 'ThorSwap',
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/bridges/thorchain.png',
+  aggregator_address: '0x7a68c37D8AFA3078f3Ad51D98eA23Fe57a8Ae21a',
+  lim: '100001618',
+  affiliate: 'thor1h2hf0txuzdl2hedx9epcqc6zf65aldk4ckpwcw',
+  fee: 10
+}
+
+//TODO: edit this
+export const LIFI: Bridge = {
+  id: 'lifi',
+  name: 'LI.FI',
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/bridges/lifi.png',
+  aggregator_address: '0x7a68c37D8AFA3078f3Ad51D98eA23Fe57a8Ae21a',
+  lim: '100001618',
+  affiliate: 'thor1h2hf0txuzdl2hedx9epcqc6zf65aldk4ckpwcw',
```

Comment:
looks like we are not using these fields. if you want to keep these fields lets make it empty or we can remove it if not needed and later if needed we can use it back

Reply:
fixed